sklearn/datasets/_svmlight_format_io.py

```diff
@@ -268,7 +268,8 @@ def load_svmlight_files(
 
     multilabel : bool, default=False
         Samples may have several labels each (see
-        https://www.csie.ntu.edu.tw/~cjlin/libsvmtools/datasets/multilabel.html)
+        https://www.csie.ntu.edu.tw/~cjlin/libsvmtools/datasets/multilabel.html
+        ).
```

Comment:
Our character limit is 88, so that `)` should fit. Did you get an error here from the tests or a linter?

Reply:
Thank you for the reviews! No, I cut the line out of habit for 80 (oops). Let me fix that.